src/loc/cs-cz.ts

```diff
@@ -0,0 +1,421 @@
+    "TeamChannelPickerFontIconFavoriteText": "Oblíbené",
+    "TeamChannelPickerFontIconPrivateChannelTitle": "Privátní kanál",
+    "TeamChannelPickerSugestionHeaderText": "Doporučené kanály",
+    "TeamPickerButtonRemoveTitle": "odstranit",
+    "TeamPickernoResultsFoundText": "Týmy nenalezeny",
+    "TeamPickerSugestionsHeaderText": "Doporučené týmy",
+    "TeamsChannelPickerButtonRemoveTitle": "odstranit",
+    "TeamsChannelPickerNoresultsFoundText": "Kanály nenalezeny",
+
+    "ViewMore": "Zobrazit více",
+
+    "DynamicFormLoading": "Načítání...",
+    "DynamicFormPleaseWait": "Počkejte...",
+    "DynamicFormRequiredErrorMessage": "Toto pole nemůže zůstat prázdné.",
+    "DynamicFormTermPanelTitle": "Vybrat výraz",
+    "DynamicFormEnterURLPlaceholder": "Zadejte URL",
+    "DynamicFormEnterDescriptionPlaceholder": "Alternativní popis",
+    "DynamicFormDialogValidationErrorTitle": "Chyba validace",
+    "DynamicFormDialogValidationErrorMessage": "Formulář obsahuje chyby validace. Před uložením je potřeba tyto chyby odstranit.",
+    "DynamicFormNumberValueMustBeGreaterThan": "Hodnota musí být větší než {0}",
+    "DynamicFormNumberValueMustBeBetween": "Hodnota musí být mezi {0} a {1}",
+    "DynamicFormNumberValueMustBeLowerThan": "Hodnota musí být menší než {0}",
+    "customDisplayName": "Použít toto umístění:",
+    "ListItemCommentDIalogDeleteSubText": "Opravdu chcete odstranit tento komentář?",
+    "ListItemCommentsDialogDeleteTitle": "Potvrdit odstranění komentáře",
+    "ListItemCommentsLabel": "Komentáře",
+    "ListItemCommentsNoCommentsLabel": "Žádné komentáře",
+    "OrgAssetsLinkLabel": "Vaše organizace",
+
+    "ModernTaxonomyPickerDefaultPlaceHolder": "Zapište výraz pro označení",
+    "ModernTaxonomyPickerTreeTitleSingle": "Vybrat výraz",
+    "ModernTaxonomyPickerTreeTitleMulti": "Vyberte jeden nebo více výrazů",
+    "ModernTaxonomyPickerAddTagButtonTooltip": "Přidat výraz",
+    "ModernTaxonomyPickerApplyButtonText": "Použít",
+    "ModernTaxonomyPickerCancelButtonText": "Zrušit",
+    "ModernTaxonomyPickerLoadMoreText": "Načíst více",
+    "ModernTaxonomyPickerRemoveButtonText": "Odstranit",
+    "ModernTaxonomyPickerPanelCloseButtonText": "Zavřít",
+    "ModernTaxonomyPickerNoResultsFound": "Žádné výsledky",
+    "ModernTaxonomyPickerSuggestionInLabel": "v",
+
+    "ModernAudioPlay": "Přehrát",
+    "ModernAudioPause": "Pozastavit",
+    "ModernAudioIncVol": "Přidat hlasitost",
+    "ModernAudioDecVol": "Ubrat hlasitost",
+    "ModernAudioMute": "Ztlumit",
+
+    "ProgressStepsIndicatorNoSteps": "Žádné kroky",
+    "TermSertNaviagtionErrorMessage": "Výraz nebyl nalezen",
+    "TermSetNavigationNoTerms": "Výrazy nejsou definovány",
+
+    "UploadFilesButtonLabel": "Přidat",
+    "UpLoadFilesDeleteButtonLabel": "Odstranit",
+    "UpLoadFilesDragDropLabel": "Přetáhněte soubory sem, nebo klikněte na tlačítko Přidat",
+    "UpLoadFilesSelectAllLabel": "Vybrat vše",
+    "UploadFilesUploadButtonLabel": "Nahrát",
```

Comment:
There's a missing location prop regarding a new control added recently :

* `HoverReactionBarSearchEmojiPlaceholder`

You can find it [here](https://github.com/pnp/sp-dev-fx-controls-react/blob/dev/src/loc/en-us.ts#L429)